Development/src/components/FilterPanel.js

```diff
@@ -74,14 +81,23 @@ export const BooleanFilter = ({
 };
 
 export const NumberFilter = ({
+    defaultValue,
     source,
     label,
     filter,
     setFilter,
     autoFocus,
     ...props
 }) => {
-    const [value, setValue] = useState(filter[source] ? filter[source] : '');
+    const [value, setValue] = useState(() => {
+        if (defaultValue) {
```

Comment:
@RufusUtt, it seems backwards to me to prefer the `defaultState`? I thought this should be:
```javascript
if (filter[source]) {
    return filter[source];
} else if (defaultValue) {
    return defaultValue;
} else {
    return '';
}
```
which I think can be written more succinctly as:
```javascript
return filter[source] || defaultValue || '';
```

But I've probably missed something!

Reply:
I've made this tweak (apart from using special treatment for `NaN`).
I also fixed `RateFilter` which I'd broken while refactoring the original PR 🙈